src/dotnet/devcontainer-feature.json

```diff
@@ -1,35 +1,33 @@
 {
     "id": "dotnet",
-    "version": "1.1.4",
+    "version": "2.0.0",
     "name": "Dotnet CLI",
     "documentationURL": "https://github.com/devcontainers/features/tree/main/src/dotnet",
-    "description": "Installs the .NET CLI. Provides option of installing sdk or runtime, and option of versions to install. Uses latest version of .NET sdk as defaults to install.",
+    "description": "Installs .NET SDK, which includes the .NET CLI and the shared runtime. Offers options to install multiple SDK versions.",
     "options": {
         "version": {
             "type": "string",
             "proposals": [
                 "latest",
-                "7",
-                "6",
-                "3.1"
+                "lts",
+                "8.0",
+                "7.0",
+                "6.0"
             ],
             "default": "latest",
-            "description": "Select or enter a dotnet CLI version. (Available versions may vary by Linux distribution.)"
+            "description": "Version of .NET SDK to install. Use 'latest' for the latest version, 'lts' for the latest LTS version, 'X.Y' or 'X.Y.Z' for a specific version."
         },
-        "runtimeOnly": {
-            "type": "boolean",
-            "default": false,
-            "description": "Install just the dotnet runtime if true, and sdk if false."
-        },
-        "installUsingApt": {
-            "type": "boolean",
-            "default": true,
-            "description": "If true, it installs using apt instead of the release URL"
+        "additionalVersions": {
+            "type": "string",
```

Comment:
@samruddhikhandale is it possible to change this typing to suggest an array instead of a string? I discovered there is an automatic conversion from array to comma-separated strings.

This works
``` json
{
  "additionalVersions": "6.0, 5.0, 3.1"
}
```

But this also works and feels more intuitive.
``` json
{
  "additionalVersions": ["6.0", "5.0", "3.1"]
}
```

Reply:
Seems it's currently not officially supported by the spec

<img width="502" alt="image" src="https://github.com/devcontainers/features/assets/1583241/9d36c0bd-5226-46f3-913e-844eaa5db4c7">

https://github.com/devcontainers/spec/blob/main/docs/specs/devcontainer-features.md#the-options-property

The fact that an array _does_ work might be accidental?

Reply:
> But this also works and feels more intuitive.
``` json
{
  "additionalVersions": ["6.0", "5.0", "3.1"]
}
```

I do like the idea of having the `value` as an `array` and it's good to know that it works out of the box.

> Seems it's currently not officially supported by the spec

I am not confident as of why we currently don't support `array` as a type. However, one thing I can think of --> It would definitely need some tweaking to the Dev Containers extension for the `Add dev container Configuration files...` command (for accepting values in the form of an array). See 👇 

![image](https://github.com/devcontainers/features/assets/24955913/5f6528a0-1874-44e7-ba56-3cbd4164c2aa)

@joshspicer any thoughts?

Reply:
Where's the automatic conversion taking place? That's interesting

Reply:
Good question 🤷‍♂️, I analyzed the call stack starting from here
https://github.com/devcontainers/cli/blob/413c5f790f3a4972bf6f05b71d71ac946c87bc7c/src/spec-node/containerFeatures.ts#L300

I didn't find any explicit conversions or clues that hint at an implicit conversion